percona/controller/pgbackup/controller.go

```diff
@@ -265,6 +274,21 @@ func (r *PGBackupReconciler) Reconcile(ctx context.Context, request reconcile.Re
 
 		return reconcile.Result{}, nil
 	case v2.BackupSucceeded:
+		job, err := findBackupJob(ctx, r.Client, pgBackup)
+		if err == nil && len(job.Finalizers) > 0 {
```

Comment:
Why don't we assert that the specific finalizer we created for this PR, If it is set, and instead we are checking if the job finalizers are not empty? If we decide at some point to have more finalizers for some reason, what would happen to this logic?

Reply:
https://github.com/percona/percona-postgresql-operator/pull/1134/commits/3e3831927501b9d6602d394ff1a8ca126c053968